CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ What's changed
 * Make `type` property required on the semantic convention group (behind `--future` flag). ([#611](https://github.com/open-telemetry/weaver/pull/611) by @lmolkova).
 * Exclude attributes declared with `code_generation.exclude` annotations in `semconv_attributes` and other JQ attribute helpers. ([#662](https://github.com/open-telemetry/weaver/pull/662) by @lmolkova)
 * Sort metrics by name in all JQ helpers. ([#573](https://github.com/open-telemetry/weaver/issues/573) by @lmolkova)
+* Add shell completion functionality. ([#682](https://github.com/open-telemetry/weaver/pull/682) by @larrys)
```

Comment:
0.14.0 was released so this line needs to be moved up to the top under a `## Next` heading please.

Reply:
Ah, I missed that when I rebased my branch. Thanks for pointing it out.